### PR TITLE
:sparkles:  Initial Release limitations doc

### PR DESCRIPTION
**Pull request by ezrasilvera**
## Related issue(s)

Fixes #

### CODE REVIEW

File: docs/content/v0.20/release-notes.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Release notes
+
+The following sections list the known issues for each release. The issue list is not differential (i.e., compared to previous releases) but a full list representing the overall state of the specific release. 
+
+## Known constraints and limitations release: v0.20-alpha.2
+* Dynamic changes to WECs are not supported. Existing placements will not be updated when new WECs are added or when labels are added/deleted on existing WECs
+* Removing of WorkStatus objects (on the transport namespace) is not supported and may not result in recreation of that object
+* Singleton: It's the user responsibility to make sure there are no shared objects in two different (singleton) placements that target two different WECs. Currently there is no enforcement on on that. 
+* Objects on two different WDSs shouldn't have the exact same identifier (same group, version, kind, name and namespace). Such a conflict is currently not identified.
+
```
Comment (pdettori) [inline docs/content/v0.20/release-notes.md:10]: There are also some limitations related to removing labels from a workload object and changes when controller is down that are listed in https://github.com/kubestellar/kubestellar/issues/1694

**Comment by effi-ofer** [comment]
Might it be worth while to add an indication on when a particular limitation would be lifted? Some of these limitations (dynamic WEC comes to mind) are going to be very short lived.

File: docs/content/v0.20/release-notes.md (hunk 1 above)
Comment (ezrasilvera) [reply docs/content/v0.20/release-notes.md:14]: Added. Thanks

**Pull request merged**